src/components/__tests__/__snapshots__/Highlight.test.js.snap

```diff
@@ -4,25 +4,22 @@ exports[`<Highlight /> snapshots renders correctly 1`] = `
+    data-style="{\\"--plain-backgroundColor\\":\\"#2a2734\\",\\"--plain-color\\":\\"#9a86fd\\",\\"--comment-color\\":\\"#6c6783\\",\\"--prolog-color\\":\\"#6c6783\\",\\"--doctype-color\\":\\"#6c6783\\",\\"--cdata-color\\":\\"#6c6783\\",\\"--punctuation-color\\":\\"#6c6783\\",\\"--namespace-opacity\\":0.7,\\"--tag-color\\":\\"#e09142\\",\\"--operator-color\\":\\"#e09142\\",\\"--number-color\\":\\"#e09142\\",\\"--property-color\\":\\"#9a86fd\\",\\"--function-color\\":\\"#9a86fd\\",\\"--tag-id-color\\":\\"#eeebff\\",\\"--selector-color\\":\\"#eeebff\\",\\"--atrule-id-color\\":\\"#eeebff\\",\\"--attr-name-color\\":\\"#c4b9fe\\",\\"--boolean-color\\":\\"#ffcc99\\",\\"--string-color\\":\\"#ffcc99\\",\\"--entity-color\\":\\"#ffcc99\\",\\"--url-color\\":\\"#ffcc99\\",\\"--attr-value-color\\":\\"#ffcc99\\",\\"--keyword-color\\":\\"#ffcc99\\",\\"--control-color\\":\\"#ffcc99\\",\\"--directive-color\\":\\"#ffcc99\\",\\"--unit-color\\":\\"#ffcc99\\",\\"--statement-color\\":\\"#ffcc99\\",\\"--regex-color\\":\\"#ffcc99\\",\\"--atrule-color\\":\\"#ffcc99\\",\\"--placeholder-color\\":\\"#ffcc99\\",\\"--variable-color\\":\\"#ffcc99\\",\\"--deleted-textDecorationLine\\":\\"line-through\\",\\"--inserted-textDecorationLine\\":\\"underline\\",\\"--italic-fontStyle\\":\\"italic\\",\\"--important-fontWeight\\":\\"bold\\",\\"--bold-fontWeight\\":\\"bold\\",\\"--important-color\\":\\"#c4b9fe\\",\\"backgroundColor\\":\\"var(--plain-backgroundColor)\\",\\"color\\":\\"var(--plain-color)\\"}"
+    style="--plain-backgroundColor: #2a2734; --plain-color: #9a86fd; --comment-color: #6c6783; --prolog-color: #6c6783; --doctype-color: #6c6783; --cdata-color: #6c6783; --punctuation-color: #6c6783; --namespace-opacity: 0.7; --tag-color: #e09142; --operator-color: #e09142; --number-color: #e09142; --property-color: #9a86fd; --function-color: #9a86fd; --tag-id-color: #eeebff; --selector-color: #eeebff; --atrule-id-color: #eeebff; --attr-name-color: #c4b9fe; --boolean-color: #ffcc99; --string-color: #ffcc99; --entity-color: #ffcc99; --url-color: #ffcc99; --attr-value-color: #ffcc99; --keyword-color: #ffcc99; --control-color: #ffcc99; --directive-color: #ffcc99; --unit-color: #ffcc99; --statement-color: #ffcc99; --regex-color: #ffcc99; --atrule-color: #ffcc99; --placeholder-color: #ffcc99; --variable-color: #ffcc99; --deleted-textDecorationLine: line-through; --inserted-textDecorationLine: underline; --italic-fontStyle: italic; --important-fontWeight: bold; --bold-fontWeight: bold; --important-color: #c4b9fe;"
   >
     <div
       class="token-line"
-      data-style="{\\"backgroundColor\\":null,\\"color\\":\\"#9a86fd\\"}"
-      style="color: rgb(154, 134, 253);"
+      data-style="{\\"backgroundColor\\":null,\\"color\\":\\"var(--plain-color)\\"}"
     >
       <span
         class="token punctuation"
-        data-style="{\\"color\\":\\"#6c6783\\"}"
-        style="color: rgb(108, 103, 131);"
+        data-style="{\\"color\\":\\"var(--punctuation-color)\\"}"
```

Comment:
This snapshot is incorrect. We didn't actually delete the `style` attribute from this span like the diff suggests. `react-test-renderrer` just flips out when you include CSS variable references like `var(--my-variable)` and decides not to render the `style` attribute. If you inspect the DOM in a working example, you'll see the `style` attribute.